perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -113,7 +120,13 @@ def _RunPing(sending_vm, receiving_vm, receiving_ip, ip_type):
   stats = re.findall('([0-9]*\\.[0-9]*)', stdout.splitlines()[-1])
   assert len(stats) == len(METRICS), stats
   results = []
-  metadata = {'ip_type': ip_type,
+
+  if ip_type == vm_util.IpAddressSubset.INTERNAL:
+    ip_string = 'internal'
```

Comment:
Can you unify this with the constants in netperf_benchmark?  Defining them in vm_util seems reasonable for now.

Reply:
I've made a new enum in vm_util and changed netperf,iperf,ping to use it